RequestManagementSystem/private/OperationHandlerBase.py

```diff
@@ -147,91 +148,91 @@ def setOperation( self, operation ):
 #     return cls.__dataLoggingClient
 
   @classmethod
-  def rssClient( cls ):
+  def rssClient(cls):
     """ ResourceStatusClient getter """
     if not cls.__rssClient:
       from DIRAC.ResourceStatusSystem.Client.ResourceStatus import ResourceStatus
       cls.__rssClient = ResourceStatus()
     return cls.__rssClient
 
-  def getProxyForLFN( self, lfn ):
+  def getProxyForLFN(self, lfn):
     """ get proxy for lfn
 
     :param str lfn: LFN
     :return: S_ERROR or S_OK( "/path/to/proxy/file" )
     """
-    dirMeta = returnSingleResult( self.fc.getDirectoryMetadata( os.path.dirname( lfn ) ) )
+    dirMeta = returnSingleResult(self.fc.getDirectoryMetadata(os.path.dirname(lfn)))
     if not dirMeta["OK"]:
       return dirMeta
     dirMeta = dirMeta["Value"]
 
-    ownerRole = "/%s" % dirMeta["OwnerRole"] if not dirMeta["OwnerRole"].startswith( "/" ) else dirMeta["OwnerRole"]
+    ownerRole = "/%s" % dirMeta["OwnerRole"] if not dirMeta["OwnerRole"].startswith("/") else dirMeta["OwnerRole"]
     ownerDN = dirMeta["OwnerDN"]
 
     ownerProxy = None
-    for ownerGroup in getGroupsWithVOMSAttribute( ownerRole ):
-      vomsProxy = gProxyManager.downloadVOMSProxy( ownerDN, ownerGroup, limited = True,
-                                                   requiredVOMSAttribute = ownerRole )
+    for ownerGroup in getGroupsWithVOMSAttribute(ownerRole):
+      vomsProxy = gProxyManager.downloadVOMSProxy(ownerDN, ownerGroup, limited=True,
+                                                  requiredVOMSAttribute=ownerRole)
       if not vomsProxy["OK"]:
-        self.log.debug( "getProxyForLFN: failed to get VOMS proxy for %s role=%s: %s" % ( ownerDN,
-                                                                                          ownerRole,
-                                                                                          vomsProxy["Message"] ) )
+        self.log.debug("getProxyForLFN: failed to get VOMS proxy for %s role=%s: %s" % (ownerDN,
+                                                                                        ownerRole,
+                                                                                        vomsProxy["Message"]))
         continue
       ownerProxy = vomsProxy["Value"]
-      self.log.debug( "getProxyForLFN: got proxy for %s@%s [%s]" % ( ownerDN, ownerGroup, ownerRole ) )
+      self.log.debug("getProxyForLFN: got proxy for %s@%s [%s]" % (ownerDN, ownerGroup, ownerRole))
       break
 
     if not ownerProxy:
-      return S_ERROR( "Unable to get owner proxy" )
+      return S_ERROR("Unable to get owner proxy")
 
     dumpToFile = ownerProxy.dumpAllToFile()
     if not dumpToFile["OK"]:
-      self.log.error( "getProxyForLFN: error dumping proxy to file: %s" % dumpToFile["Message"] )
+      self.log.debug("getProxyForLFN: error dumping proxy to file: %s" % dumpToFile["Message"])
```

Comment:
Isn't it a bit violent?

Reply:
It's reported at the higher level already, and the rest of the messages in that method are already debug